src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -126,56 +126,46 @@ ParticipantConfiguration::ParticipantConfiguration(
+                                        "This participant needs to provide the mesh using `<provide-mesh />`.");
+
+  auto attrDirectAccess = makeXMLAttribute(ATTR_DIRECT_ACCESS, false)
                               .setDocumentation(
-                                  "If a mesh is received from another partipant (see tag <from>), it needs to be"
-                                  "decomposed at the receiving participant. To speed up this process, "
-                                  "a geometric filter (see tag <geometric-filter>), i.e. filtering by bounding boxes around the local mesh, can be used. "
-                                  "This safety factor defines by which factor this local information is "
-                                  "increased. An example: 0.5 means that the bounding box is 150% of its original size.");
-  tagReceiveMesh.addAttribute(attrSafetyFactor);
+                                  "Controls direct access to a received mesh without having to map it to a provided mesh. "
+                                  "A received mesh needs to be decomposed using a region of interest, which cannot inferred, if there are no mappings to or from a provided mesh. "
+                                  "To manually provide the region use the API function `setMeshAccessRegion()`.");
+  tagReceiveMesh.addAttribute(attrDirectAccess);
 
   auto attrGeoFilter = XMLAttribute<std::string>(ATTR_GEOMETRIC_FILTER)
                            .setDocumentation(
-                               "If a mesh is received from another partipant (see tag <from>), it needs to be"
-                               "decomposed at the receiving participant. To speed up this process, "
-                               "a geometric filter, i.e. filtering by bounding boxes around the local mesh, can be used. "
-                               "Two different variants are implemented: a filter \"on-primary\" strategy, "
-                               "which is beneficial for a huge mesh and a low number of processors, and a filter "
-                               "\"on-secondary\" strategy, which performs better for a very high number of "
-                               "processors. Both result in the same distribution (if the safety factor is sufficiently large). "
-                               "\"on-primary\" is not supported if you use two-level initialization. "
-                               "For very asymmetric cases, the filter can also be switched off completely (\"no-filter\").")
+                               "A received mesh needs to be decomposed using a mapping-inferred region of interest, if it is not directly-accessed. "
+                               "A geometric filter based on bounding-boxes around the local mesh can speed up this process. "
+                               "`on-primary-rank` is beneficial for a huge mesh and a low number of processors, but is incompatible with two-level initialization. "
```

Comment:
Do we need to mention that this is where the filter is applied?

Reply:
Yes, good point.

Reply:
```suggestion
                               "This setting configures if and where this fitler is applied. "
                               "`on-primary-rank` is beneficial for a huge mesh and a low number of processors, but is incompatible with two-level initialization. "
```

Reply:
Done